hladny_matfyzak.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import hladnymatfyzak
+
+
+DATE_FORMAT = '%d.%m.%Y'
```

Comment:
I am only OK with this only per @Adman's assurance that it will be refactored for errbot.